Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -5,20 +5,20 @@
 #include <algorithm>
 #include <cstdio>
 #include <memory>
+#include <utility>
 
 #include "Common/CommonTypes.h"
 #include "Common/FileUtil.h"
 #include "DiscIO/CISOBlob.h"
 
 namespace DiscIO
 {
-static const char CISO_MAGIC[] = "CISO";
-
-CISOFileReader::CISOFileReader(std::FILE* file) : m_file(file)
+CISOFileReader::CISOFileReader(File::IOFile&& file) : m_file(std::move(file))
```

Comment:
> But I still have to use...

Yes

> Wouldn't it be problematic to use std::move while taking by value if someone makes IOFile copyable in the future?

No, because an interface that's already built around moves will still work. Just as moving a std::string doesn't have adverse effects when it's no longer needed in a current scope, despite having a copy-constructor.